NuRadioReco/modules/analogToDigitalConverter.py

```diff
@@ -237,65 +222,51 @@ def get_digital_trace(self, station, det, channel,
         """
 
         station_id = station.get_id()
-        if channel_id is None or not trigger_adc:
-            channel_id = channel.get_id()
-
+        channel_id = channel.get_id()
         det_channel = det.get_channel(station_id, channel_id)
 
+        field_prefix = 'trigger_' if trigger_adc else ''
```

Comment:
Does this need to be so compressed?

Reply:
I agree to that in some extend. I think there is a reason to use some neat python syntax over making the code the most easiest to read. I am happy to change it here. (Although in a new PR I am moving this code to a private function and I do not think that newcomers have to read this function.)